.github/workflows/integration-tests.yml

```diff
@@ -1,6 +1,53 @@
 name: Integration tests
 on:
   workflow_dispatch:
+    inputs:
+      database1:
+        description: 'Database (v1) - a new one will be created if not provided'
+        required: false
+        default: ''
+      database2:
+        description: 'Database (v2) - a new one will be created if not provided'
+        required: false
+        default: ''
+      database-with-engine-v2:
+        description: 'Database (v2) for Firebolt v2 - a new one will be created if not provided'
+        required: false
+        default: ''
+      engine1:
+        description: 'Engine (v1) - a new one will be created if not provided'
+        required: false
+      engine2:
+        description: 'Engine (v2) - a new one will be created if not provided'
+        required: false
+      engine_v2_fb_2_0:
+        description: 'Engine (v2) for Firebolt v2 - a new one will be created if not provided'
+        required: false
```

Comment:
OK, I agree, but I'd still leave the choice elements here to be at least able to choose specific environments only.

Reply:
There is only one problem now. Did yo pay attention that I pass even execution string using parameter instead of passing "good" category? This is done because in previous version the command looked like the following:

```
dotnet test --filter 'Category=Integration&Category=v2&(Category!=v1|Category=!=engine-v2)' -l "console;verbosity=normal"
```

This actually means that additionally to passing "good" category I have to pass "bad" categories. So, I decided to pass the full command line instead. Well, I could pass the filter only, probably this is better. 

I had to stop using this technique because in this case I cannot mark test for both `v2` and `engine-v2`.


So, if you have better solution for tests filtering/categorizing please suggest it here or in the previous discussion. Once this is done we will be able to fix the parameters issue.

Reply:
Yet another idea. Instead of supplying database, engine and run:true/false separately we can do this for together for each environment:

```
      run-v5:
        description: 'Run tests against Firebolt DB v 5: db:engine'
        required: true
        default: 'default:default'
```

Empty value means "do not run"
Any other value contains DB and Engine names separated by colon. The split can be done in script. The world "default" means "use default database/engine". 

This solution is more scalable because it requires only one parameter per enviromnent and therefore can support up to 10 environments but the script will have to split string and the configuration will be a little bit more error prone.

Reply:
I removed configuration from the main integration-tests.yml, so it runs test with difault configuration. Proprietary scripts integration-tests-v1 and integration-tests-v2 can be executed either from the main script or directly; in this case the configuration can be given manually. I hope now it meets your wishes.